packages/expo-clipboard/android/src/test/java/expo/modules/clipboard/ClipboardModuleTest.kt

```diff
@@ -54,15 +54,16 @@ class ClipboardModuleTest {
   //   assertEquals("album dumbledore", readResult)
   // }
 
-  @Test
-  fun `should get empty string when clipboard is empty`() = withClipboardMock {
-    // This requires API 28
-    clipboardManager.clearPrimaryClip()
-
-    val content = module.getStringAsync()
-
-    assertTrue("Clipboard content should be empty", content.isEmpty())
-  }
+  // TODO (barthap): Uncomment this once fixed race condition "React Application Context is null"
```

Comment:
cc @barthap: looks like there are even more broken tests 😞

Reply:
yet another failures at the `
expo.modules.clipboard.ClipboardModuleTest > should throw when ClipboardManager is unavailable FAILED`. 😅